docs/features/secrets/general.md

```diff
@@ -60,4 +60,13 @@ After a hard refresh you can use the latest secret again and proceed your work.
 Secret secret = await cachedSecretProvider.GetSecretAsync("EventGrid-AuthKey", ignoreCache: true);
 ```
 
+## Invalidates a secret from the cach
+In some scenarios you'd like to remove the entry from the cache, so the next the secret is retrieved, a new lookup will be done in the provider.
+
+This is useful for scenario's where the secret is updated and you need to tell the cache somehow.
+
+```csharp
+await cachedSecretProvider.InvalidateSecretAsync("EventGrid-AuthKey");
```

Comment:
`InvalidateSecretAsync` or `InvalidateAsync`? But I presume we have `GetSecretAsync`?

```suggestion
await cachedSecretProvider.InvalidateAsync("EventGrid-AuthKey");
```

Reply:
It's `CachedSecretProvider.GetSecretAsync(string)` so it's ok.

Reply:
Yeah, that's why I added the `...Secret...`. part